clinica/pipelines/pet_surface/pet_surface_utils.py

```diff
@@ -200,10 +200,60 @@ def isclose(a, b, rel_tol=1e-9, abs_tol=0.0):
     return list_of_regions
 
 
+def runApplyInverseDeformationField_SPM_standalone(target,
+                                                   deformation_field,
+                                                   img,
+                                                   matscript_folder):
+    """
+    This function does the exact same job as runApplyInverseDeformationField but with SPM standalone. We directly create
+    a batch file that SPM standalone can run. This function does not check wether SPM standalone must be used. Previous
+    check when building the pipeline ensures that all the env vars exists ($SPMSTANDALONE_HOME and $MCR_HOME)
+    """
+    from os.path import abspath, join, basename, exists
+    import platform
+    import os
+
+    prefix = 'subject_space_'
+
+    # Write SPM batch command directly in a script that is readable by SPM standalone
+    script_location = abspath('./m_script.m')
+    script_file = open(script_location, 'w+')
+    script_file.write('jobs{1}.spm.util.defs.comp{1}.inv.comp{1}.def = {\'' + deformation_field + '\'};\n')
+    script_file.write('jobs{1}.spm.util.defs.comp{1}.inv.space = {\'' + target + '\'};\n')
+    script_file.write('jobs{1}.spm.util.defs.out{1}.pull.fnames = {\'' + img + '\'};\n')
+    script_file.write('jobs{1}.spm.util.defs.out{1}.pull.savedir.saveusr = {\'' + abspath(os.getcwd()) + '\'};\n')
+    script_file.write('jobs{1}.spm.util.defs.out{1}.pull.interp = 4;\n')
+    script_file.write('jobs{1}.spm.util.defs.out{1}.pull.mask = 1;\n')
+    script_file.write('jobs{1}.spm.util.defs.out{1}.pull.fwhm = [0 0 0];\n')
+    script_file.write('jobs{1}.spm.util.defs.out{1}.pull.prefix = \'' + prefix + '\';\n')
+    script_file.close()
+
+    # Generate command line to run
+    # SPM standalone must be run directly from its root folder
+    if platform.system() == 'Darwin':
+        # Mac OS
+        cmdline = 'cd $SPMSTANDALONE_HOME && ./run_spm12.sh $MCR_HOME batch ' + script_location
+    elif platform.system() == 'Linux':
+        # Linux OS
+        cmdline = '$SPMSTANDALONE_HOME/run_spm12.sh $MCR_HOME batch ' + script_location
+    else:
+        raise SystemError('Only support Mac OS and Linux')
+    os.system(cmdline)
+
+    output_file = join(abspath('./'), prefix + basename(img))
+    if not exists(output_file):
+        raise IOError('Something went wrong while trying to run runApplyInverseDeformationField_SPM_standalone'
+                      + '. Output file not generated. Command launched :\n\t '
+                      + cmdline + '\n. We strongly recommand that you use the supported version of Matlab MCR '
+                      + ' recommanded by the creators of SPM.')
+    return output_file
```

Comment:
I have no expertise for the `runApplyInverseDeformationField_SPM_standalone` function. If you want a feedack, I would advise you to ask @mdiazmel :-)